docs/source/examples/workspaces/openstack-ocp/ocp-run.yml

```diff
@@ -0,0 +1,13 @@
+#jinja2:lstrip_blocks: True
+---
+libvirt-network:
```

Comment:
Why are you creating an openstack server and a libvirt node?

Reply:
For the libvirt testing